curriculum/challenges/chinese/02-javascript-algorithms-and-data-structures/es6/create-strings-using-template-literals.chinese.md

```diff
@@ -7,7 +7,7 @@ localeTitle: 使用模板文字创建字符串
 ---
 
 ## Description
-<section id="description"> ES6的一个新功能是<dfn>模板文字</dfn> 。这是一种特殊类型的字符串，可以更轻松地创建复杂字符串。模板文字允许您创建多行字符串并使用字符串插值功能来创建字符串。考虑以下代码： <blockquote> const person = { <br>名称：“Zodiac Hasbro”， <br>年龄：56岁<br> }; <br><br> //具有多行和字符串插值的模板文字<br> const greeting =`您好，我的名字是$ {person.name}！ <br>我是$ {person.age}岁。 <br><br>的console.log（问候）; //打印<br> //你好，我的名字是Zodiac Hasbro！ <br> //我今年56岁<br></blockquote>那里发生了很多事情。首先，例如使用反引号（ <code>`</code> ），而不是引号（ <code>&#39;</code>或<code>&quot;</code> ），换行字符串。其次，请注意，该字符串是多线，无论是在代码和输出。这节省了插入<code>\n</code>串内。上面使用的<code>${variable}</code>语法是占位符。基本上，您不必再使用<code>+</code>运算符连接。要将变量添加到字符串，只需将变量放在模板字符串中并用<code>${</code>包装它<code>}</code>同样，您可以在您的字符串表达式的其他文字，例如<code>${a + b}</code>这个新创建的字符串的方式为您提供了更大的灵活性，以创建强大的字符串。 </section>
```

Comment:
Aren't there still some breaking glyphs here, at the line starts?

Reply:
@yi-ge Can you respond to @lynxlynxlynx's question?